splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -1686,6 +1686,10 @@
           "type": "string",
           "description": "Version of UCC used during build process"
         },
+        "hideUCCVersion": {
+          "type": "string",
```

Comment:
Can we have it as boolean with a default value?
Also, where should we document this?

Reply:
+1 regarding boolean question.

regarding documentation - I'll create a ticket to create a documentation page for `meta` fields.

Reply:
good catch!